Source/Mod.cs

```diff
@@ -14,6 +13,7 @@ public class Mod : Verse.Mod
     {
         public Mod( ModContentPack content ) : base( content )
         {
+            GetSettings<Settings>();
```

Comment:
This is needed just to initialize private `Mod.modSettings` variable. If it's `null`, `Mod.WriteSettings` does nothing.